src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -186,6 +364,28 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityProps> = ({
     )
   }, [])
 
+  const poolAssetInboundAddress = useMemo(() => {
+    if (!poolAsset) return
+    const transactionType = getThorchainLpTransactionType(poolAsset.chainId)
+
+    switch (transactionType) {
+      case 'MsgDeposit': {
+        return THORCHAIN_POOL_MODULE_ADDRESS
+      }
+      case 'EvmCustomTx': {
+        // TODO: this should really be inboundAddressData?.router, but useQuoteEstimatedFeesQuery doesn't yet handle contract calls
+        // for the purpose of naively assuming a send, using the inbound address instead of the router is fine
+        return inboundAddressesData?.address
+      }
+      case 'Send': {
+        return inboundAddressesData?.address
+      }
+      default: {
+        assertUnreachable(transactionType as never)
+      }
```

Comment:
the braces around switch cases are normally to allow defining new variables within a case, so shouldn;'t be required here

Reply:
That's fair - fixed in https://github.com/shapeshift/web/pull/6175/commits/55d61881b96485d99aa59c851bf113e41c291381.